app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -24,30 +30,58 @@
     </div>
     <div v-if="delegation.loaded && undelegatedValidators.length > 0">
```

Comment:
Should we get rid of this section now?

Reply:
Yeah the initial idea was to just drop this and replace with the transactions, then I left to keep the discussion open on whether we want or not a kind of grouping per validator node for the u-transactions